packages/synapse-react-client/src/components/widgets/query-filter/EnumFacetFilter/EnumFacetFilter.tsx

```diff
@@ -137,8 +156,19 @@ function _EnumFacetFilter(props: EnumFacetFilterProps) {
     )
     const valueNotSetFacetArray = partitions[0]
     const restOfFacetValuesArray = partitions[1]
+    let sortedValues: RenderedFacetValue[]
+    if (isNumberColumnType) {
+      sortedValues = sortBy(restOfFacetValuesArray, fv => new Number(fv.value))
```

Comment:
Any reason to not use `Number(...)` which returns a number, instead of `new Number(...)`, which returns an object?
```suggestion
      sortedValues = sortBy(restOfFacetValuesArray, fv => Number(fv.value))
```